lib/bottom_navy_bar.dart

```diff
@@ -217,4 +223,7 @@ class BottomNavyBarItem {
   /// This will take effect only if [title] it a [Text] widget.
   final TextAlign? textAlign;
 
+  /// The [background] solid color defined when this item is selected. Defaults
+  /// to [false].
+  final bool solidBackground;
```

Comment:
Maybe change it to: If [true], use the solid color for the background of this item. Defaults to [false].